src/Entity/Form/AppEditForm.php

```diff
@@ -145,6 +146,7 @@ public function form(array $form, FormStateInterface $form_state) {
           '#title' => $api_product_def->getPluralLabel(),
           '#required' => TRUE,
           '#options' => $credential_product_options,
+          '#disabled' => $this->canEditApiProducts(),
```

Comment:
```suggestion
          '#disabled' => !$this->canEditApiProducts(),
```
This needs to be negated :stuck_out_tongue:

Reply:
Woah. Thanks @arlina-espinoza 👍